beacon-chain/blockchain/block_processing.go

```diff
@@ -70,7 +71,7 @@ func (c *ChainService) ReceiveBlock(ctx context.Context, block *pb.BeaconBlock)
 		return nil, fmt.Errorf("could not retrieve beacon state: %v", err)
 	}
 
-	blockRoot, err := hashutil.HashBeaconBlock(block)
+	blockRoot, err := blockutil.BlockSigningRoot(block)
```

Comment:
this may be out of scope for this PR,
I wonder if `SigningRoot` belongs in `blockutil` because it's more generalized that just `block`. We use `SigningRoot` for `transfer`, `slashing` and `deposit` objects as well

I think putting it in`hashutil` and renaming to `SigningRoot` are better

Reply:
I agree, but like you said for another PR.